ipaserver/install/server/install.py

```diff
@@ -336,6 +336,12 @@ def install_check(installer):
             "Please uninstall it before configuring the IPA server, "
             "using 'ipa-client-install --uninstall'")
 
+    instance_name = "-".join(options.realm_name.split("."))
+    if (is_ipa_configured() and
```

Comment:
Well, if the right way to check is with `if not options.external_cert_files`, I think I could put the code in line [406](https://github.com/freeipa/freeipa/blob/master/ipaserver/install/server/install.py#L406), so.

I think it makes more sense to do the check in the `install_check` method instead of in `install`, what do you think?

> I am not sure whether DS should be started in this case or not, if it were me, I would just probably have it started since it's a requirement for the installation.

In my POV, since this is a corner case, I think we should not re-start DS because this is already done in the first step. Warn/raise an error is enough to the user take some action.